gradle/verification-metadata.xml

```diff
@@ -1064,11 +1065,21 @@
             <sha256 value="05621bf96e33332f1fddb3522cd17d3a1afebefbef5d123b897f84ece0667fc6" origin="Verified" reason="Artifact is not signed"/>
          </artifact>
       </component>
+      <component group="org.asciidoctor" name="asciidoctor-gradle-base" version="4.0.0-alpha.1">
+         <artifact name="asciidoctor-gradle-base-4.0.0-alpha.1.jar">
+            <sha256 value="59281a4861a380148bfb9924abcb1287bccdb6484955ac447577cf8bde17eefc" origin="Generated by Gradle"/>
```

Comment:
:x: The `origin` should be updated, even better would be adding a `reason`. See the comments at the beginning of this file.